### ISSUE TITLE
Business details

### ISSUE THREAD

**Issue by govuk-design-system** [alphagov/govuk-design-system-backlog#7]
## What
Help users provide information about a business. Some types include:

- sole trader
- partnership
- limited liability company
- limited liability partnership
- unincorporated association
- charity
- cooperative

**Comment by soupdragon99** [comment]
# Dropbox Paper audit

On 22 February 2019 the Design System team reviewed a Dropbox Paper document discussing the Business Type pattern.

The aim was to reduce the number of places containing guidance and code by:

- migrating relevant, useful content into the Design System itself
- recording important research findings in the community backlog
- removing the original Dropbox Paper page

Below is a record of the outcomes of that review.

If you need to, you can [see the original Dropbox Paper content in the internet archive](https://web.archive.org/web/20190222130309/https://paper.dropbox.com/doc/Business-type--AYAeeLzh66mKBZEsYnT8zMPXAg-UHCpoFo1hw8MsBgP4WUL3).

## Review outcomes

### Updates to the Design System

The Design System team will carry out the following updates to ensure that relevant, useful content from the Dropbox Paper file is added to the Design System. 

- this issue is logged in the backlog and marked as to do
- draft content for this guidance added to this issue for reference

**Comment by soupdragon99** [comment]
# When to use this pattern
Not everyone knows or remembers their type of business, and the terms have changed over the years. If you can avoid asking the question then do so. Consider whether you can look the information up automatically.
## How it works
### Explain what the terms mean 
Common causes of confusion include:
self-employed people not recognising themselves as being sole traders
clubs / associations not recognising the term ‘unincorporated body'
confusion between two sole traders who divide work up between themselves, and the legal concept of Partnership
## Research on this pattern
### HMRC
 Type of business: 
Individual / soletrader
Limited company
Limited Liability Partnership
Ordinary Partnership
Unincorporated body
Non-UK company

We've  already found that people didn't understand that the term 'Corporate body' has changed to 'Limited company'. 
### Legal Aid
From Christine Elliot:
"We ask them to specify Self Employment / Business Partnership/ Shareholder and/or Company Director in a private limited company. A minority of users do get confused with the questions but generally this is not an issue and when it does happen it is resolved by writing out for clarification.
 
This is accompanied with the following help text:

**Are you self employed?**
By self employed we mean anyone who is engaged in a trade, profession or vocation as
a sole trader (i.e. you have sole ownership of the business and its profits). This includes
anyone who is required to complete a self-assessment tax return as a self employed
person. If you operate in partnership with someone else (i.e. you share the ownership of
the business and its profits with one or more other person) you should answer no to
this question and refer to question 2 below.
 
**Are you a partner in a business partnership?**
By business partnership we mean anyone who is engaged in a trade, profession or
vocation which is operated in partnership with someone else (i.e. you share the
ownership of the business and its profits with one or more other person) but the
business is not incorporated as a company. This includes anyone who is required to
complete a self-assessment tax return as a partner.

You should answer yes to this question even if you are a sleeping partner (i.e. you are
not actively involved in the partnership, but you are entitled to a share of the business
and its profits).
 
**Are you a shareholder in a private limited company and/or a company director?**
By company director we mean anyone registered as a director of an incorporated body/
company (excluding charities), whether or not they hold shares in that body/company.

**Comment by cisasearching** [comment]
Hi,
I can't find anything else on this topic other than this page. 

I'm currently involved in a project which is seeing a similar problem. Users have to say what type of business they are (between organisation, Individual provider and partnership). 

On the form, when the user selects an option, a few explanatory lines about that option is revealed. (Max length is two lines on a standard laptop monitor display)

Many users submit the wrong option here (we know this from our returns data - where we return forms with incorrect information). 
User research we have conducted (albeit in very small numbers) say when they first see the page, that they want to know what each option means. Before they click an option and see that there is help text, they say how useful it would be to have a definition on screen. 

We've developed a couple of options that we will user test soon. 

- One uses the details components (suspicion that users will miss it and not click on it)
- One retains the conditional reveal nature of the existing hint text, but worded differently
- One has all of the text on screen with the description being the same component as the option (which then makes the options quite long)
- One has the description in a help text component and it appears under each option - it is displayed at all times and is not revealed or hidden according to a user action. 

All descriptions to be a maximum of two lines long, any longer then readability and ease of understanding may decrease.

**Comment by cjforms** [comment]
Have you thought about doing this the other way around, and putting the hint text ahead of the options rather than being revealed later?
The hint text would go between the question and the answer options, like in the textarea component: https://design-system.service.gov.uk/components/textarea/

**Comment by cisasearching** [comment]
> Have you thought about doing this the other way around, and putting the hint text ahead of the options rather than being revealed later? The hint text would go between the question and the answer options, like in the textarea component: https://design-system.service.gov.uk/components/textarea/

Thanks for the comment back. I appreciate it. Yes, this was an initial option, but it felt like it made the hint text too long to read all in one go. Perhaps it is worth testing it with users too though. My personal thought is that we do want to move away from revealing descriptions later.

**Comment by cjforms** [comment]
I definitely agree that revealing descriptions when they are too late to be useful is a bad idea. Here are some suggestions:

1. (As before) Put the text as hint text. Work on getting it down to a maximum of two lines, as you suggested.
2. If you can't make it short enough to be correct and clear (and this is indeed difficult), then consider placing it behind a 'show description' or 'tell me more about this' link in the hint text, so that the person can choose to view the text before trying their answer.
3. If neither 1 nor 2 works, then you can structure the conversation of the form so that it's less annoying, like this:
- accept whatever answer they give you
- on the next page, show them a page that has their answer and the explanation along the lines of 
'You've chosen sole trader, Our definition of sole trader is [.... explanation here ....]. Is this correct for your business?
 (  ) yes, it's correct   
 (  ) no, I want to change my answer   
 (  ) not sure, please show me more definitions

The advantage of the 'next page in the flow' strategy is that you can choose whether to show that page to everyone, whatever they reply, or to limit the page to people who choose categories that you know are likely to be problematic. For example, my guess would be that people are relatively unlikely to make mistakes with 'charity' and 'limited company' and more likely to make mistakes with some of the other categories.

**Comment by cisasearching** [comment]
Yeah option 2 is already one of my options. I like the sound of your suggested number 3 though!
